ex04.ex

```diff
@@ -55,7 +55,7 @@ defmodule Ex04 do
 
   """
 
-  def min . . . "your code"
+  def min(listOfNumbers), do: reduce(listOfNumbers, &(if &1 < &2 do &1 else &2 end))
```

Comment:
## -1

This function 

1. uses `if`, which I'd like you to avoid, and
2. looks more complicated than it is.

How about splitting it out:

~~~ elixir
  def min(list), do: reduce(list, &min_of/2)
  def min_of(a,  b) when a < b, do: a
  def min_of(_a, b), do: b
~~~